charts/model-engine/templates/service_template_config_map.yaml

```diff
@@ -233,7 +233,7 @@ data:
                 - "${SQS_QUEUE_URL}"
                 {{- end }}
                 - --num-workers
-                - "${PER_WORKER}"
+                - "${CONCURRENT_REQUESTS}"
```

Comment:
i think people have been pretty confused about "per_worker", possible to add documentation as much as possible to distinguish the two fields? concurrent_requests_per_worker?

Reply:
yup, will add the documentation, probably to some other site